src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-details/rgw-bucket-details.component.ts

```diff
@@ -18,6 +18,7 @@ export class RgwBucketDetailsComponent implements OnChanges {
       this.rgwBucketService.get(this.selection.bid).subscribe((bucket: object) => {
         bucket['lock_retention_period_days'] = this.rgwBucketService.getLockDays(bucket);
         this.selection = bucket;
+        this.selection.policy = JSON.parse(this.selection.policy);
```

Comment:
Regardiing the previous comments, to show `{}` instead of `null`

```suggestion
        this.selection.policy = JSON.parse(this.selection.policy) || {};
```

Reply:
ah you are right. was so simple. i'll update the PR.